workspaces/optic-engine/tests/snapshots/interaction_diff__can_yield_unmatched_shape-2.snap

```diff
@@ -1,31 +1,5 @@
 ---
-source: workspaces/optic-engine-native/tests/interaction_diff.rs
```

Comment:
This snapshot is correct now - it's in "incompliant" interaction https://github.com/opticdev/optic/blob/develop/workspaces/optic-engine/tests/interaction_diff.rs#L251 - this should not generate a diff

Interestingly, this test has been here for a while, so this is probably something we knew about?